packages/orchestration/src/util/filtering.ts

```diff
@@ -0,0 +1,41 @@
+import type {
+  AzureContentSafety,
+  AzureContentSafetyFilterConfig,
+  InputFilteringConfig,
+  OutputFilteringConfig
+} from '../client/api/schema/index.js';
+
+/**
+ * Convenience function to create Azure content filters.
+ * @param filter - Filtering configuration for Azure filter. If skipped, the default Azure content filter configuration is used.
+ * @returns An object with the Azure filtering configuration.
+ * @deprecated Use ContentFilters.buildAzureContentSafety() instead.
+ */
+export function buildAzureContentFilter(
+  filter?: AzureContentSafety
+): InputFilteringConfig | OutputFilteringConfig {
+  return {
+    filters: [ContentFilters.buildAzureContentSafety(filter)]
+  };
+}
```

Comment:
[req] The behaviour of this convenience function has been altered, it originally used to throw an error when `filter` was empty and the associated tests have been removed too.
Please revert this change as it's a breaking change.

Reply:
Reverted behaviour in [commit](https://github.com/SAP/ai-sdk-js/pull/441/commits/77af75e448976b4722f3b7f87942aef4d4ccfe58).